storage/include/libstqe.sh

```diff
@@ -36,11 +35,12 @@ function stqe_init_fwroot
 
     # clone the framework
     typeset fwroot=$(stqe_get_fwroot)
-    rlRun "rm -rf $fwroot"
-    rlRun "git clone $STQE_GIT $fwroot" || rlAbort "fail to clone $STQE_GIT"
+    cki_run_cmd_neu "rm -rf $fwroot"
```

Comment:
Should this one really be the `_neu` variant? The original call doesn't have the `$BKRM_RC_ANY`...

Reply:
The original code was written by myself. I think it's better to remove directory $fwroot without checking its return code as we use `**rm -rf ...**`. So `cki_run_cmd_neu` is used instead of `cki_run_cmd_pos`.